### PR TITLE
implementation of creating trip api

### LINKED ISSUE #4: Implement Database Feature in CREATETRIP API
1. Finish code review
2. give database design

### PR DESCRIPTION
**Pull request by Connor-ZH**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 /.idea/
-/__pycache__/
\ No newline at end of file
+*/__pycache__/
```
Comment (Connor-ZH) [inline .gitignore]: add a new empty line

File: Dispatch/controler.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    :param driver:
+    :return: True or False (Accept or not)
+    '''
+
+def get_trip_detail(trip_id):
+    '''
+    :param trip_id:
+    :return: trip_detail=[trip_id,username,location_pickup,location_dropoff,time]
+    '''
+    pass
```
Comment (Connor-ZH) [inline Dispatch/controler.py]: add a new empty line

File: Dispatch/view.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+import uuid
+
+def create_trip(location_pickup,location_dropoff,user_name):
+    trip_id = str(uuid.uuid1()).replace('-', '')
+    controler.store_trip(trip_id,user_name,location_pickup,location_dropoff)
+    return trip_id
+
+def get_driver_id(trip_id):
+    driver_id = controler.get_driver_id(trip_id)
+    return driver_id
```
Comment (Connor-ZH) [inline Dispatch/view.py]: add a new empty line

File: User/templates/booking.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<body>
+<div id="datetime">
+    <script>
+        setInterval("document.getElementById('datetime').innerHTML=new Date().toLocaleString();", 1000);
```
Comment (Connor-ZH) [inline User/templates/booking.html]: add javascript file

File: User/templates/show_booking_result.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+<h>Hi, your trip id is {{trip_id}}</h>
+
+<body>
+    <br>
+    <button onclick="search_driver()">
+        Confirm
+    </button>
+    <br>
+      
+    <script>
```
Comment (Connor-ZH) [inline User/templates/show_booking_result.html]: add javascript file

File: User/templates/show_booking_result.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        var timer;
+        function serach_driver() {
+            // timer = setInterval(update_status(),4000);
+            var xhr = new XMLHttpRequest();
+            xhr.open('post', 'http://127.0.0.1:5002/update_trip_status',true)
```
Comment (Connor-ZH) [inline User/templates/show_booking_result.html]: 1. Save link in config.py
2. return link in render template

**Review by liwanghuan — APPROVED** [review]
LGTM

**Pull request merged**